ci/kind/kind-setup.sh

```diff
@@ -242,6 +244,32 @@ function configure_extra_networks {
   done
 }
 
+function configure_kind_ipam_vlan_subnets {
+  if [[ -z $IPAM_VLAN_SUBNETS || -z $IPAM_VLAN_IDS ]]; then
+    return
+  fi
+  
+  IFS=',' read -r -a ipam_vlan_subnets <<< "$IPAM_VLAN_SUBNETS"
+  IFS=',' read -r -a ipam_vlan_ids <<< "$IPAM_VLAN_IDS"
+  i=0
+
+  for s in "${ipam_vlan_subnets[@]}"; do
+    vlan_id=${ipam_vlan_ids[$i]}
+    i=$(($i+1))
+    echo "Configuring VLAN-$vlan_id subnets"
+    bridge_id=$(docker network inspect kind -f {{.ID}})
+    bridge_interface="br-${bridge_id:0:12}"
+    vlan_interface="br-${bridge_id:0:7}.$vlan_id"
+
+    docker_run_with_host_net ip link add link $bridge_interface name $vlan_interface type vlan id $vlan_id
+    docker_run_with_host_net ip link set $vlan_interface up
+    echo "Configuring extra IP $s to vlan interface $vlan_interface"
+    docker_run_with_host_net ip addr add dev $vlan_interface $s
+    docker_run_with_host_net iptables -t filter -A FORWARD -i $bridge_interface -o $vlan_interface -j ACCEPT
+    docker_run_with_host_net iptables -t filter -A FORWARD -o $bridge_interface -i $vlan_interface -j ACCEPT
```

Comment:
bridge interface and vlan intrface configured due to provided subnets and vlan ids can hang there on test bed even if kind cluster is deleted , shld be cleaned up after test execution

Reply:
yes,added the cleaned up code